packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -3697,12 +3697,18 @@
   "task": {
     "dynamic": false,
     "properties": {
+      "apiKey": {
+        "type": "binary"
+      },
       "attempts": {
         "type": "integer"
       },
       "enabled": {
         "type": "boolean"
       },
+      "invalidateApiKey": {
+        "type": "boolean"
+      },
```

Comment:
Q: Do we need them in the mappings? Or is it enough for the field to exist in the document but not be indexed?

Reply:
Good point, will remove those from the mapping